test/support/api/forms.js

```diff
@@ -18,20 +18,20 @@ Cypress.Commands.add('routeForms', (mutator = _.identity) => {
     .as('routeForms');
 });
 
-Cypress.Commands.add('routeForm', (mutator = _.identity, formId = '11111') => {
+Cypress.Commands.add('routeFormByAction', (mutator = _.identity, formId = '11111') => {
   cy
     .fixture('test/forms').as('fxTestForms');
 
   cy.route({
-    url: '/api/forms/*',
+    url: '/api/actions/*/form',
     response() {
       return mutator({
         data: getResource(_.find(this.fxTestForms, { id: formId }), 'forms'),
         included: [],
       });
     },
   })
-    .as('routeForm');
+    .as('routeFormByAction');
 });
```

Comment:
Is this correct?

New `routeFormByAction` command is identical to `routeForm` (now removed), apart from the URL?